chat-widget/automation_tests/e2e/utility/constants.ts

```diff
@@ -0,0 +1,107 @@
+export enum CustomLiveChatWidgetConstants {
+  CustomLiveChatWidgetFilePath = "customlivechatwidgets/CustomLiveChatWidget.html",
+  LiveChatButtomId = "#oc-lcw-chat-button-title",
+  LivechatButton="//div[@class='oclcw-t1']",
+  DefaultTimeout = 1000,
+  CustomLCWAdaptiveCardWidgetFilePath = "customlivechatwidgets/CustomLCWAdaptiveCard.html",
+  CustomLCWAdaptiveCardWrapWidgetFilePath = "customlivechatwidgets/CustomLCWAdaptiveCardWrap.html",
+  CustomLCWAttachmentPrechatWidgetFilePath = "customlivechatwidgets/CustomLcwAttachementPreChatWidget.html",
+  CustomLCWBotWidgetFilePath = "customlivechatwidgets/CustomeLCWBotChatWidget.html",
+}
+
+export enum SelectorConstants {
+  BotwaitGreetMsg = "//p[contains(text(),'Hi Customer! This is your Omnichannel Test Bot 🤖')]",
```

Comment:
please add only required selector constants for added test cases.

Reply:
remove not required constants

Reply:
All the declared selector constants are using in spec file of test cases.